stmhal/usbd_conf.c

```diff
@@ -1,587 +1,654 @@
-/*

```

Comment:
It looks like you changed the line endings or something to cause every line to be different. I think you should preserve the line endings so that we can see the changes. If we're going to change line endings, that should be in a commit by itself.